lara-typescript/src/page-item-authoring/managed-interactives/customize.tsx

```diff
@@ -88,6 +89,13 @@ export const CustomizeManagedInteractive: React.FC<Props> = (props) => {
             name={formField("linked_interactive_id").name}
             defaultValue={`${linked_interactive_id || ""}`}
           />
+          <select
+            name={formField("linked_interactive_type").name}
+            defaultValue={`${linked_interactive_type || ""}`}
+          >
+            <option value="MWInteractive">MWInteractive</option>
+            <option value="ManagedInteractive">ManagedInteractive</option>
+          </select>
```

Comment:
This will be tough for authors. Here are a couple of alternatives:
- use PageItem ids: this requires a migration and updating the ids that are shown at the top of the authoring dialog to be the PageItem id instead of the embeddable id
- add the type to the id that is shown in the authoring dialog so it is something like MWInteractive_1234 or ManagedInteractive_1234. And then update the controller which handles the linked_interactive_id form so it parses this id into its two parts. And the defaultValue of the linked_interactive_id would have to be changed too.

Do you have a preference between the two?

Reply:
I thought about using the page item ids but I don't think we surface that in the UI anywere for authors.  Would it be better to rename the display for the options to "Traditional Interactive" and "Managed Library Interactive"?

Reply:
I think no mater what we call it, they will have problems. Most of them won't really care about what type of interactive they are linking too. In both of my 2 options we have to change the id that is shown at the top of the authoring dialog. The only we show this id is people can copy it when linking interactives. So it is fine to change this id to either a PageItem or a string that includes the type.